src/utils/useLocalStorage.js

```diff
@@ -1,7 +1,7 @@
-export const SaveLocalStorage = (key, value) => {
-  localStorage.setItem(key, value)
+export const setLocalStorage = (key, value) => {
+  return localStorage.setItem(key, JSON.stringify(value))
 }
 
-export const SaveJsonLocalStorage = (key, value) => {
-  localStorage.setItem(key, JSON.stringify(value))
+export const getLocalStorage = (key) => {
```

Comment:
I think getFromLocalStorage is a better name

Reply:
Agree 👍🏻 , variable name has been changed